llvm/test/CodeGen/SPIRV/function/internal-anonymous-function.ll

```diff
@@ -0,0 +1,20 @@
+; RUN: llc -O0 -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s
+
+; Types:
+; CHECK-DAG: [[F32:%.+]] = OpTypeFloat 32
```

Comment:
In addition to Andrey's suggestion, we use a different register designation (example form opencl/image.ll):
```
; CHECK: %[[#A:]] = OpFunctionParameter %[[#IMG_1D]]
...
; CHECK: %[[#PIXEL:]] = OpImageRead %[[#VEC:]] %[[#A]] %[[#]]
```
So it would better to correct this too.